frontend/packages/console-dynamic-plugin-sdk/src/utils/fetch/console-fetch.ts

```diff
@@ -32,8 +32,11 @@ const consoleFetchCommon = async (
   method: string = 'GET',
   options: RequestInit = {},
   timeout?: number,
+  isFullResponse?: boolean,
 ) => {
   const headers = getConsoleRequestHeaders();
+  const defaultIsFullResponse = typeof isFullResponse !== 'boolean' ? false : isFullResponse;
```

Comment:
I would skip the `typeof` check and just use `isFullResponse` directly. If someone isn't using TypeScript and passes any truthy value, we should treat it as true.

Reply:
> I would skip the `typeof` check and just use `isFullResponse` directly. If someone isn't using TypeScript and passes any truthy value, we should treat it as true.

@spadgett Hmm, I tried skipping this but it broke at least one `K8sCreate` method instance response returned after `.then` which should be  `.data` for `fullRrsponse` in the console. It seems the `K8sCreate` instance passed unused property -`metadata.name` to `consoleFetch` which caused the issue. I can figure out and correct that along the line but added the `typeof` to  enforce the type and increase our confidence that this wouldn't cause any breaking change upstream.

Reply:
> Are you saying `k8sCreate` is passing an extra parameter to `consoleFetch`? That sounds like a bug we should understand and track down.

@spadgett Yes, I think so. See attached for error and log for create pod action without the type check:
The `isFullResponse` logs `example5` which is the `metadata.name` of the pod. I think it is a bug too. 
<img width="1110" alt="Screenshot 2024-03-04 at 9 58 17 AM" src="https://github.com/openshift/console/assets/15249132/24a96835-4273-4301-83aa-c8d9d4b18ac0">

Reply:
Let's open a bug to track.